.github/ISSUE_TEMPLATE/service-proposal.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Service proposal
+about: Template to use to submit a service proposal
+title: "[SERVICE]"
+labels: service
+assignees: ''
+
+---
+
+<!-- 
+
+Use the following template when submitting proposals for new CUSTOM services to this repository.
+
+ Linking to a standard characteristic/service specification document on the Bluetooth SIG's website is sufficient for standard characteristic/service proposals. When proposing standard services, also include specific requirement information for the Mbed BLE implementation of this service (eg: memory constraints, baremetal support, etc) 
+
+The following site can be used to generate a long-format UUID for custom services/characteristics: https://www.uuidgenerator.net/
+This site is useful for generating markdown-formatted tables: https://www.tablesgenerator.com/markdown_tables
+-->
+
+### BLE Standard Service: Yes/No
```

Comment:
maybe we can use a tickbox `[ ]` fro that

Reply:
That's a good point.